tests/components/samsungtv/test_config_flow.py

```diff
@@ -797,14 +762,15 @@ async def test_ssdp_websocket_cannot_connect(hass: HomeAssistant) -> None:
 
 
 @pytest.mark.usefixtures("remote")
-async def test_ssdp_model_not_supported(hass: HomeAssistant) -> None:
+async def test_ssdp_wrong_manufacturer(hass: HomeAssistant) -> None:
     """Test starting a flow from discovery."""
-
+    ssdp_data = deepcopy(MOCK_SSDP_DATA)
+    ssdp_data.upnp[ATTR_UPNP_MANUFACTURER] = ssdp_data.upnp[ATTR_UPNP_MANUFACTURER][7:]
```

Comment:
I think the test name was misleading...
The config flow does not rejected the device based on "model" information, but based on "manufacturer" information

Again, since we don't have a "real-life" fixture, I think it is clearer to re-use the default fixture and tweak inside the test